Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions-details.tsx

```diff
@@ -142,40 +125,13 @@ const SubmissionsDetailsProvider = ({ children }: ISubmissionsDetailsProviderPro
             return;
         }
 
-        setCurrentProblemSubmissionResults(apiProblemResults);
-    }, [ apiProblemResults ]);
-
-    const {
-        get: getProblemResultsByUserRequest,
-        data: apiProblemResultsByUser,
-    } = useHttp({
-        url: getSubmissionResultsByProblemAndUserUrl,
-        parameters: submissionResultsByProblemAndUserUrlParams,
-    });
-
-    useEffect(
-        () => {
-            if (isNil(submissionResultsByProblemAndUserUrlParams)) {
-                return;
-            }
-
-            (async () => {
-                startLoading();
-                await getProblemResultsByUserRequest();
-                setSubmissionResultsByProblemAndUserUrlParams(null);
-                stopLoading();
-            })();
-        },
-        [ getProblemResultsByUserRequest, startLoading, stopLoading, submissionResultsByProblemAndUserUrlParams ],
-    );
-
-    useEffect(() => {
-        if (isNil(apiProblemResultsByUser)) {
+        if (!isNil(apiProblemResultsError)) {
```

Comment:
Why not expose apiProblemResultsError directly?

Reply:
Great suggestion! It is done.